### PR TITLE
chore: Make team area translatable (add t-var)

### LINKED ISSUE #5794: Teams Permission Page: Make all areas translatable
Seems like the teams permission page of events is not fully translatable yet.

Screenshot from 2020-11-27 21-02-49

### PR DESCRIPTION
**Pull request by SushantAA**
Fixes #5794 

#### Short description of what this resolves:
Adds translation to Teams permission page
T-var add in permission.title

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: app/templates/events/view/team/permissions.hbs [hunk 1]
```diff
@@ -12,7 +12,7 @@
         {{#if permission.header}}
           <td colspan="3" class="left aligned" style="background-color: rgba(0, 0, 0, 0.05);">{{permission.title}}</td>
```
Comment (MeghalBisht) [inline app/templates/events/view/team/permissions.hbs:13]: Here also

Comment (SushantAA) [reply app/templates/events/view/team/permissions.hbs]: done

**Review by MeghalBisht — APPROVED** [review]

**Comment by iamareebjamal** [comment]
Why are your sentences and clauses reversed? translation make? t-var add? Yoda?

**Comment by SushantAA** [comment]
I will take care of that :upside_down_face:

**Pull request merged**